navigation/index.tsx

```diff
@@ -39,14 +44,58 @@ function ProfileStackScreen() {
   )
 }
 
+const CamNav = createNativeStackNavigator();
+
+function CameraStackNav()
+{
+  return (
+    <CamNav.Navigator
+      
+    >
+
+    <CamNav.Screen
```

Comment:
Please format the indentation. You can google how to format the code.
Like, on VSC, you can use shift+option(alt)+f to format the code.

Reply:
Sounds good, again my apologies